/home/ubuntu/Imview_server/avenv/lib/python3.4/site-packages/flask_sqlalchemy/__init__.py:800: UserWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True to suppress this warning.
  warnings.warn('SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True to suppress this warning.')
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
127.0.0.1 - - [15/Dec/2015 12:15:43] "GET / HTTP/1.0" 200 -
127.0.0.1 - - [15/Dec/2015 12:15:52] "GET /image/list?is_featured=1 HTTP/1.0" 200 -
127.0.0.1 - - [15/Dec/2015 12:15:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:15:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:16:02] "GET /image/list?is_featured=1 HTTP/1.0" 200 -
127.0.0.1 - - [15/Dec/2015 12:16:05] "GET /image/list?is_featured=1 HTTP/1.0" 200 -
127.0.0.1 - - [15/Dec/2015 12:16:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:16:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:16:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:16:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:17:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:17:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:17:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:17:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:18:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:18:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:18:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:18:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:19:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:19:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:19:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:19:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:20:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:20:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:20:43] "GET / HTTP/1.0" 200 -
127.0.0.1 - - [15/Dec/2015 12:20:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:20:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:21:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:21:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:21:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:21:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:22:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:22:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:22:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:22:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:23:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:23:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:23:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:23:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:24:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:24:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:24:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:24:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:25:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:25:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:25:43] "GET / HTTP/1.0" 200 -
127.0.0.1 - - [15/Dec/2015 12:25:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:25:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:26:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:26:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:26:38] "GET /image/list?is_featured=1 HTTP/1.0" 200 -
Traceback (most recent call last):
  File "/home/ubuntu/Imview_server/app/views.py", line 86, in add_new_image
    file = request.files['file']
  File "/home/ubuntu/Imview_server/avenv/lib/python3.4/site-packages/werkzeug/datastructures.py", line 402, in __getitem__
    raise exceptions.BadRequestKeyError(key)
werkzeug.exceptions.BadRequestKeyError: 400: Bad Request
127.0.0.1 - - [15/Dec/2015 12:26:49] "POST /image/new HTTP/1.0" 400 -
127.0.0.1 - - [15/Dec/2015 12:26:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:26:54] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:27:23] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:27:24] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:27:53] "GET /app.php HTTP/1.0" 404 -
127.0.0.1 - - [15/Dec/2015 12:27:54] "GET /app.php HTTP/1.0" 404 -
